ui-v2/app/helpers/object-entries.js

```diff
@@ -0,0 +1,7 @@
+import { helper } from '@ember/component/helper';
+
+export function objectEntries([obj = {}] /*, hash*/) {
```

Comment:
I wasn't really sure why the additional need for this helper was but I assume it is to transform the shape of the returned addresses into something suitable to render.

Reply:
Again, great assumption, and I couldn't have explained it better myself (no surprises there! 😛 )

Here's the reference doc:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

It transforms:

```javascript
{
  "lan": "something",
  "wan": "else"
}
```

to

```javascript
[
  ["lan", "something"],
  ["wan", "else"]
]
```

...as the `tabular-collection` component takes an array of values not an object.

This helper just wraps the javascript implementation so we can use it within handlebars.